mlflow/tensorflow.py

```diff
@@ -1059,16 +1033,11 @@ def __init__(self):
         def _patch_implementation(
             self, original, inst, *args, **kwargs
         ):  # pylint: disable=arguments-differ
-            active_run = mlflow.active_run()
-            if MLFLOW_AUTOLOGGING in active_run.data.tags:
-                global _AUTOLOG_RUN_ID
-                _AUTOLOG_RUN_ID = active_run.info.run_id
-
```

Comment:
This logic is unnecessary, as `FitPatch` applies to tf.keras models, while `_AUTOLOG_RUN_ID` is only used for model tracking during `tf.estimator.Estimator` model serialization (tf.keras models aren't instances of tf.estimator.Estimator).